californium-core/src/main/java/org/eclipse/californium/core/server/resources/ResourceAttributes.java

```diff
@@ -251,6 +251,34 @@ public void clearObservable() {
 		attributes.remove(LinkFormat.OBSERVABLE);
 	}
 
+	/**
+	 * Marks the resource as only accessible using OSCORE.
+	 *
+	 * @since 3.10
+	 */
+	public void setOscoreOnly() {
+		findAttributeValues(LinkFormat.OSCOREONLY).setOnly("");
+	}
+
+	/**
+	 * Checks if the resource is only accessible using OSCORE.
+	 *
+	 * @since 3.10
```

Comment:
please exchange the order of since and return.

Reply:
I have fixed this also now